src/tuner/nccl_ofi_tuner.c

```diff
@@ -352,6 +352,8 @@ ncclResult_t nccl_ofi_tuner_get_coll_info(void *context,
 		protocol = nccl_ofi_tuner_ctx->regions[i].protocol;
 		if (table[algorithm][protocol] == NCCL_ALGO_PROTO_IGNORE || algorithm >= numAlgo ||
 		    protocol >= numProto) {
```

Comment:
Not related to this PR, but shouldn't the check for `algorithm >= numAlgo` come before trying to access `table[algorithm]`? Same for protocol.

Reply:
Yes, good catch! I fixed it